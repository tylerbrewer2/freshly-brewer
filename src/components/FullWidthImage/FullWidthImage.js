import React from 'react';
import styled from 'styled-components';

const FullWidthImage = styled.div`
  background-image: url(${props => props.src})
  background-size: cover;
  background-position: center;
  height: 800px;
`

export default FullWidthImage;