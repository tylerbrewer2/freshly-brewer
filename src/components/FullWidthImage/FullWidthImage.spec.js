import React from 'react';
import renderer from 'react-test-renderer';

import FullWidthImage from './FullWidthImage';

it('renders correctly', () => {
  const tree = renderer.create(
    <FullWidthImage />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
