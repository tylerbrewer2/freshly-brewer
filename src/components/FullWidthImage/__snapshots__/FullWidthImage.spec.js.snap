exports[`test renders correctly 1`] = `
<div
  className="sc-bdVaJa eHbHUx" />
`;
